scylla/src/cluster/metadata.rs

```diff
@@ -601,7 +601,8 @@ impl MetadataReader {
     async fn fetch_metadata(&self, initial: bool) -> Result<Metadata, MetadataError> {
         // TODO: Timeouts?
         self.control_connection.wait_until_initialized().await;
-        let conn = ControlConnection::new(self.control_connection.random_connection()?);
+        let conn = ControlConnection::new(self.control_connection.random_connection()?)
+            .override_timeout(None);
```

Comment:
Commit "metadata: customise ControlConnection's request timeout"

Why call `.override_timeout(None)` here? It is none even without this call.

Reply:
It's there just for explicitness. The `None` serves as a placeholder, which is replaced by an actual value in the next commit.